crates/holochain/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Fixes bug where supplying a `network_seed` during an `InstallApp` call does not actually update the network seed for roles whose `provisioning` is set to `None` in the manifest. Now the network seed is correctly updated. [\#2102](https://github.com/holochain/holochain/pull/2102)
 
+- Adds new functionality to the conductor admin API which returns disk storage information. The storage used by apps
+  is broken down into blobs which are being used by one or more app.
+
```

Comment:
Hey @ThetaSinner was this meant to be part of 0.2.0-beta-rc.2 or UNRELEASED?

Reply:
Should be under unreleased, I must have done a bad rebase. I'll fix this now